includes/wikia/VariablesBase.php

```diff
@@ -5152,6 +5152,18 @@
  */
 $wgJsMimeType = 'text/javascript';
 
+/**
```

Comment:
I would put these assignments [somewhere here](https://github.com/Wikia/app/blob/dev/LocalSettings.php#L43), closer to other `getenv()` and leave only simple declarations in `VariablesBase.php` (no function calls, no logic in this file). Ideally, I would like to keep this file injecting identical state in any environment. `getenv()` here makes it implicitly env-specific.